docgen/generate-docs.js

```diff
@@ -323,7 +323,9 @@ Promise.all([
     // Clean up temp home markdown file. (Nothing needs to wait for this.)
     fs.unlink(tempHomePath);
     // Devsite doesn't like css.map files.
-    return fs.unlink(`${docPath}/assets/css/main.css.map`);
+    if (fs.existsSync(`${docPath}/assets/css/main.css.map`)) { 
+      return fs.unlink(`${docPath}/assets/css/main.css.map`);
```

Comment:
When I run this script, I don't get any css.map file generated by Typedoc, and so the script errors out. It's possible that the correct solution here is just to get rid of this unlink entirely, but I figured this was slightly safer in case I'm missing some reason that css.map would get generated .

Reply:
I did some digging and I can't find out why either. LGTM.